denylist/Cargo.toml

```diff
@@ -21,5 +21,6 @@ serde = { workspace = true }
 serde_json = { workspace = true }
 config = { workspace = true }
 chrono = { workspace = true }
-xorf-generator = { git = "https://github.com/helium/xorf-generator", branch = "main" }
 humantime-serde = { workspace = true }
+
+xorf-generator = { git = "https://github.com/helium/xorf-generator", branch = "mj/helium-crypto-update" }
```

Comment:
Should it be updated to the `main` branch?

Reply:
The branches for `helium-lib` and `xorf-generator` will be merged in their respective repos and updated here before this PR is merged; similar to a proto update.